File: BackEnd/src/main/java/lk/ijse/gdse/pos/entity/OrderDetail.java
package lk.ijse.gdse.pos.entity;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import org.modelmapper.internal.bytebuddy.implementation.bind.annotation.Super;

@AllArgsConstructor
@NoArgsConstructor
@Data
public class OrderDetail implements SuperEntity {
    private String oId;
    private String itemCode;
    private double unitPrice;
    private int qty;
   // private double total;
}

File: FrontEnd/model/OrderDetail.js
export class OrderDetail {
    get orderId() {
        return this._orderId;
    }

    set orderId(value) {
        this._orderId = value;
    }

    get itemCode() {
        return this._itemCode;
    }

    set itemCode(value) {
        this._itemCode = value;
    }

    get description() {
        return this._description;
    }

    set description(value) {
        this._description = value;
    }

    get qty() {
        return this._qty;
    }

    set qty(value) {
        this._qty = value;
    }

    get unitPrice() {
        return this._unitPrice;
    }

    set unitPrice(value) {
        this._unitPrice = value;
    }
    constructor(orderId, itemCode, description, qty, unitPrice) {
        this._orderId = orderId;
        this._itemCode = itemCode;
        this._description = description;
        this._qty = qty;
        this._unitPrice = unitPrice;
    }


}